### PR TITLE
Fix nginx YAML format

### PR DESCRIPTION
**Pull request by chillu**

### CODE REVIEW

File: platform/configs/nginx.yaml [hunk 1]
```diff
@@ -70,7 +70,7 @@ nginx::nginx_locations:
     vhost: "mysite"
     ssl: true
     www_root: "/var/www/html"
-    location: "~* /\."
+    location: "~* \\."
```
Comment (Firesphere) [inline platform/configs/nginx.yaml]: I think it still needs the forward slash as well?

Comment (brettt89) [reply platform/configs/nginx.yaml]: Yeah the fix here is to use single quotes not double

File: platform/configs/nginx.yaml [hunk 2]
```diff
@@ -70,7 +70,7 @@ nginx::nginx_locations:
     vhost: "mysite"
     ssl: true
     www_root: "/var/www/html"
-    location: "~* /\."
+    location: '~* \.'
```
Comment (brettt89) [inline platform/configs/nginx.yaml]: Add forward slash back into rule

Comment (brettt89) [reply platform/configs/nginx.yaml]: it should match /. for any request (e.g. /.gitignore) Otherwise it will match any URL that contains a dot.

**Comment by chillu** [comment]
Double quotes require more escaping in YAML. Check the `mysite_media_nginx` a bit further down for comparison

**Pull request merged**